src/main/java/uk/gov/companieshouse/web/lfp/security/WebSecurity.java

```diff
@@ -25,8 +25,20 @@ protected void configure(HttpSecurity http) throws Exception {
         }
     }
 
+
     @Configuration
     @Order(2)
+    public static class AccessibilityStatementPageSecurityConfig extends WebSecurityConfigurerAdapter {
+
+        @Override
+        protected void configure(HttpSecurity http) throws Exception {
+            http
+                    .antMatcher("/late-filing-penalty/accessibility-statement");
```

Comment:
Could this be added to line 24 to make 

`.antMatcher("/late-filing-penalty").antMatcher("/late-filing-penalty/accessibility-statement");`

?

That method would need renaming from `TemporaryStartPageSecurityConfig` to something like `StartPagesNoLogInRequired`

Reply:
I did try that, looks like the last usage of the .antMatcher method is the one used when built.

Reply:
Fair enough. Guess it'll have to do

Reply:
It's different because the home page is meant to be hosted externally...and every other URL in the footer is hosted externally.